wikibase/1.33/bundle/Dockerfile

```diff
@@ -11,14 +11,16 @@ bash download-extension.sh Elastica;\
 bash download-extension.sh CirrusSearch;\
 bash download-extension.sh UniversalLanguageSelector;\
 bash download-extension.sh cldr;\
-# Temporary, will remove once 1.34 is there
-curl -s "https://gerrit.wikimedia.org/r/plugins/gitiles/mediawiki/extensions/EntitySchema/+archive/99ec5bcaef2fdcc6ffa4ea9ca49b47c95ea76a0f.tar.gz" -o EntitySchema.tar.gz;\
 tar xzf OAuth.tar.gz;\
 tar xzf Elastica.tar.gz;\
 tar xzf CirrusSearch.tar.gz;\
 tar xzf UniversalLanguageSelector.tar.gz;\
 tar xzf cldr.tar.gz;\
 tar xzf WikibaseImport.tar.gz;\
+rm ./*.tar.gz
```

Comment:
this isn't needed as we are deleting everything on line 26, or?

Reply:
In line 23 we are making a new tar.gz file again :D (EntitySchema.tar.gz) I can make it explicitly delete everything.

Reply:
I don't get it .. whatever tar.gz files that you make/download before line 26 will be deleted once you reach line 26 .. so line 20 seems redundant